.circleci/config.yml

```diff
@@ -31,6 +31,10 @@ jobs:
       # run tests!
       - run: ./gradlew test
 
+      # run test coverage report!
+      - run: ./gradlew build jacocoTestReport
```

Comment:
the report task can be run at the end of testing and there is no need to run the `build` task.

the run command from above can be modified to say:
`-run: ./gradlew test jacocoTestReport`

and then this line can be removed

Reply:
Thanks